pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1376,7 +1376,8 @@ private void createPersistentTopic(final String topic, boolean createIfMissing,
         }
 
         if (isTransactionSystemTopic(topicName)) {
-            String msg = String.format("Can not create transaction system topic %s", topic);
+            String msg = String.format("Only has metadata, used for transactionCoordinator load balance, "
+                    + "never need to be created, for transaction system topic %s", topic);
```

Comment:
Ah...it's funny.
@Anonymitaet Could you help review this?

Reply:
@liangyepianzhou Sorry, I don't quite understand the meaning. Could you please explain a bit more?

Reply:
Of course, the current situation is that we have some system topics for lookup that only need to create metadata. 
Then the user used admin to query this topic, but our error made the user confused and raised the issue.

Reply:
Ignore creation of  transaction system topic, no need to create.